### DISCUSSION TITLE
Deye SUN-10k-G05 works?

### DISCUSSION THREAD

**Discussion by VAC91** [StephanJoubert/home_assistant_solarman#509] [Q&A]
I am interested in buying Deye SUN-10k-G05. Will it work with this integration? What YAML file should I use?

**Comment by PV-Joe** [comment]
I have the Deye Sun 8K EU and it works with the version 1.4..3. Did not update to the latest version as this is described to be faulty.

I used the Deye_SG04LP3.yaml

**Reply by VAC91** [reply]
But Deye_SG04LP3.yaml is for hybrid inverters. Do you really have G05 version? 10K-G05 isn't hybrid inverter.

**Comment by PV-Joe** [comment]
OK, yes mine is Hybrid, sorry.
